.github/workflows/build-system-test.yml

```diff
@@ -0,0 +1,146 @@
+# Description: this workflow runs build mega-apps that have all the connected components.
+
+name: Build System Test Canary
+
+permissions:
+  id-token: write # This is required for aws-actions/configure-aws-credentials
+
+on:
+  schedule:
+    - cron: '*/15 * * * *' # Run every 15 minutes
```

Comment:
Only trigger every 15 min for now. 

We'll add to publish-next and publish-latest